### DISCUSSION TITLE
Workspace runnables

### DISCUSSION THREAD

**Discussion by adsick** [helix-editor/helix#10229] [General]
Hi, it would be awesome to be able to pick a binary to run/debug from the current workspace (maybe it is there but I didn't find)

also little feedback:
btw today looked at Helix after some time and I like how I've downloaded ```rust-analyzer```, put it into ```PATH``` and it just works with Helix! with my previous (neo)Vim setup I struggled a lot with setup, obscure-looking lua scripts and in the end you just realize that [you can't switch between diagnostics to see what's wrong](https://github.com/simrat39/rust-tools.nvim/issues/100)
Helix seems a lot more appealing to beginners although I didn't learned some things (like jump history navigation which allows to jump back after go to definition)... oh, wait, it's just ```Ctrl-o``` and ```Ctrl-i``` then I can call myself a power-user!

**Comment by Omnikar** [comment]
As for debugging, see #574.
